docs/commands/index.md

```diff
@@ -21,6 +21,21 @@ slate theme [name]
 
 Generate a new blank theme. The `name` argument is required for the new theme directory.
 
+##### Options
+```
+--yarn  install theme dependencies with yarn instead of npm
+```
+
+> **Note about Yarn:** While Yarn is not the default installer for dependencies, it is worth noting that Yarn has significant performance improvements and we do recommend installing dependencies via `--yarn`.
+>
+> The primary concepts behind Yarn are that:
```

Comment:
I don't feel it's necessary to explain this list.  The sentence before is enough of an endorsement for the docs.  
Could you please add a link to https://yarnpkg.com/en/docs/install so folks know how to install it - they can read more about the benefits there.